vocab/credentials/v2/vocabulary.yml

```diff
@@ -23,6 +23,16 @@ class:
     deprecated: true
     comment: A type of validator that can be used to syntactically validate JSON documents using the JSON Schema language.
 
+  - id: JsonSchema2023
+    label: JSON schema validator 2023
+    upper_value: cred:CredentialSchema
+    comment: A type of validator that can be used to syntactically validate JSON documents using the JSON Schema language.
+
+  - id: CredentialSchema2023
+    label: Credential schema validator 2023
+    upper_value: cred:CredentialSchema
+    comment: A type of validator that can be used to syntactically validate JSON documents packaged as Verifiable Credentials using the JSON Schema language.
```

Comment:
```suggestion
    comment: A type of validator that can be used to syntactically validate JSON documents packaged as Verifiable Credentials using the JSON Schema language.
```

Reply:
won't let me apply this one for some reason 🤔

Reply:
Something must have overlapped... Anyway, looks right, now.